torchinductor/codegen/wrapper.py

```diff
@@ -70,6 +70,10 @@ def __init__(self):
                 aten = torch.ops.aten
             """
         )
+        if config.triton.use_mm:
```

Comment:
Would it be better if override codegen of MatrixMultiplication class
```
if self.kernel == "aten.mm.out":
   wrapper_writeline("...")
elif self.kernel == "triton_mm_out":
   wrapper_writeline("...")
```
Because if we find inputs type is not cuda, we won't import triton ops in this. case.

Reply:
That makes sense! I think that can work too.

Reply:
Actually I just took a look, and this class WrapperCodeGen doesn't have the concept (field) of kernel, I think it's just about headers.